test/mocha/mock.data.js

```diff
@@ -0,0 +1,126 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+/* jshint node: true */
+
+'use strict';
+
+const {util: {uuid}} = require('bedrock');
+
+const data = {};
+module.exports = data;
+
+const accounts = data.accounts = {};
+const zcaps = data.zcaps = [];
+
+// regular permissions
+const email = 'alpha@example.com';
+accounts[email] = {};
+accounts[email].account = createAccount(email);
+accounts[email].meta = {};
+accounts[email].meta.sysResourceRole = [{
+  sysRole: 'bedrock-test.regular',
+  // FIXME: had to enable admin rights to create keyStore
+  // generateResource: 'id'
```

Comment:
Do things break if this is enabled?  I found in other places that issues related to this have been resolved.  If possible, uncommenting this and removing fixme would be good.

Reply:
Can't enable just yet. I am getting a Permission Error.

https://gist.github.com/gannan08/c5fd30274052ca40f1d29c389eb9d45c